service/src/main/java/bio/terra/workspace/service/workspace/WorkspaceService.java

```diff
@@ -369,10 +368,12 @@ public String enablePetServiceAccountImpersonation(
     try {
       Policy saPolicy =
           crlService.getIamCow().projects().serviceAccounts().getIamPolicy(petSaName).execute();
+      // TODO(PF-991): In the future, the pet SA should not be included in this binding. This is a
+      //  workaround to support Nextflow and other applications which call Pipelines API.
```

Comment:
nit: the CLI and other applications which call the pipelines api with the pet SA's credentials.
Just noting that this restriction isn't really a problem with nextflow -- nextflow can be called with the end user's credentials. it's the CLI's quirky way of calling nextflow via a docker container that's the problem.